streamtasks/src/main/java/com/leverx/task/service/Impl/TaskDTOServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package com.leverx.task.service.Impl;
+
+import com.leverx.task.entity.Task;
+import com.leverx.task.entity.TaskDTO;
+import com.leverx.task.service.TaskDTOService;
+
+import java.util.List;
+
+public class TaskDTOServiceImpl implements TaskDTOService {
+
+    public TaskDTO convert(Task task) {
+        return new TaskDTO(task.getId(), task.getTitle(), task.getType(), task.getCreatedOn());
+    }
+
+    public Object[] convertTasksToTaskDTO(List<Task> tasks) {
+        return tasks.stream().map(task -> new TaskDTOServiceImpl().convert(task)).toArray();
+    }
```

Comment:
Don't create a new Service every time and also don't return an object array
A little bit improved examples:
```
public TaskDTO[] convertTasksToTaskDTO(List<Task> tasks) {
        return tasks.stream().map(task -> convert(task)).toArray(TaskDTO[]::new);
    }
```
or
```
  public List<TaskDTO> convertTasksToTaskDTO(List<Task> tasks) {
        return tasks.stream().map(task ->convert(task)).collect(toList());
    }
```